azure/functions/decorators/function_app.py

```diff
@@ -1373,3 +1375,92 @@ def decorator():
             return decorator()
 
         return wrap
+
+    def event_grid_trigger_message(self,
```

Comment:
to be consistent with other decorators, can you change to event_grid_trigger?

Reply:
good catch. I missed seeing this,